gneiss/regression/_mixedlm.py

```diff
@@ -260,3 +271,27 @@ def _format(x):
         smry.add_df(scores, align='r')
 
         return smry
+
+
+# q2 cli
+
+
+def lme_regression(table: pd.DataFrame, tree: skbio.TreeNode,
+                   metadata: pd.DataFrame, formula: str,
+                   groups: str) -> LMEModel:
+    res = mixedlm(table=table, tree=tree, metadata=metadata,
+                  formula=formula, groups=groups)
+    res.fit()
+    return res
+
+
+plugin.methods.register_function(
```

Comment:
...the downside to not just depending on q2 is that bugs like this can slip through very easily. `plugin` is a `NameError` here.